src/app/(post-login)/submission/program/[shortName]/clinical-data/page.tsx

```diff
@@ -18,18 +18,211 @@
  */
 'use client';
 
+import { ClinicalEntitySearchResultsQuery } from '@/__generated__/graphql';
 import { pageWithPermissions } from '@/app/components/Page';
+import { BreadcrumbTitle, HelpLink, PageHeader } from '@/app/components/PageHeader/PageHeader';
+import CLINICAL_ENTITY_SEARCH_RESULTS_QUERY from '@/app/gql/CLINICAL_ENTITY_SEARCH_RESULTS_QUERY';
+import SUBMITTED_DATA_SIDE_MENU_QUERY from '@/app/gql/SUBMITTED_DATA_SIDE_MENU_QUERY';
+import useUrlParamState from '@/app/hooks/useUrlParamState';
+import { notNull } from '@/global/utils';
+import { useQuery } from '@apollo/client';
+import { Loader } from '@icgc-argo/uikit';
+import { useEffect, useState } from 'react';
+import { setConfiguration } from 'react-grid-system';
+import SearchBar from './SearchBar';
+import {
+	ClinicalEntitySearchResultResponse,
+	CompletionStates,
+	TsvDownloadIds,
+	defaultClinicalEntityFilters,
+	emptyClinicalDataResponse,
+	emptySearchResponse,
+	parseDonorIdString,
+	reverseLookUpEntityAlias,
+} from './common';
 
-const ClinicalDataPageComp = () => {
-	return <div>clinical data</div>;
+setConfiguration({ gutterWidth: 9 });
+
+const defaultClinicalEntityTab = 'donor';
+
+// convert codegen automated types to definitions in existing code
+// make data object shape uniform
+const parseSearchResult = (
+	data: ClinicalEntitySearchResultsQuery | undefined,
+): ClinicalEntitySearchResultResponse => {
+	if (!data) return emptySearchResponse;
+	return {
+		clinicalSearchResults: {
+			searchResults: data.clinicalSearchResults.searchResults.map((searchResults) => ({
+				donorId: searchResults?.donorId || 0,
+				submitterDonorId: searchResults?.submitterDonorId || '',
+			})),
+			totalResults: data.clinicalSearchResults.totalResults,
+		},
+	};
+};
```

Comment:
we are using the gql codegen that was added to `platform-ui` but there are also typings that we wrote, which differ slightly from the generated types, most optional vs non-optional which cascades all kinds of falsy checks.

this is a function to manage that complexity. give the "downstream" functions what they are expecting instead of changing types in a hundred places.